### PR TITLE
add functionality to grouper-to-github.py and a lot to the readme

### PR DESCRIPTION
**Pull request by paciorek**
I added various functionality to the script: 
  
-  warning messages
-  do not create team if not requested
-  allow additional users specified individually by calnet name

I also added info to the README on how the CalGroup can be created, how the github.b.e org is requested and what to tell students to do to activate their accounts.

@ryanlovett please take a look and decide if it makes sense to merge in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +53,24 @@ Secrets file:
...
 `grouper-to-github.py -c ./config.json -s ./secrets.json`
+
+## Issues to consider
+
+We now have a flag, -u, for `grouper-to-github.py` that allows addition of users not in the CalGroup based on their CalNet username.
+
+As Ryan pointed out, it would be more coherent to add people to the CalGroup, e.g.:
+
+```
+dumbledore:~/proj/sis/sis-cli$ sis people -y 2019 -s fall -n <course_id> -i campus-uid -c enrolled | tee 2019-fall-<course_id>-enrolled-uid.txt
```
Comment (ryanlovett) [inline README.md]: Here and below, can you omit the hostname and prompt?

File: grouper-to-github.py [hunk 2]
```diff
@@ -49,14 +50,34 @@ def grouper_get_group_members(auth, group):
...
 
+def gh_account_check(auth, user):
+    '''Check if user's github.b.e account is active (i.e., have the logged in to github.b.e).'''
+    uri = '{}/users/{}'.format(config['github_base_uri'], user)
+    r = requests.get(uri, auth=auth)
+    if r.status_code == 404:
+        return False
+    else:
+        return True
+
```
Comment (ryanlovett) [inline grouper-to-github.py]: Might be simpler to `return r.status_code != 404`.

**Review by ryanlovett — APPROVED** [review]
Just a couple of suggestions.

File: grouper-to-github.py (hunk 2 above)
Comment (paciorek) [reply grouper-to-github.py]: Thanks; good points. Funny, the verbosity of that return is something I would generally point out in looking over student code ...

**Pull request merged**